src/unittests/loaders_and_trees.spec.ts

```diff
@@ -99,13 +99,17 @@ const specsData: { [k: string]: SpecFile } = {
 };
 
 class FakeTestFileLoader extends TestFileLoader {
-  async listing(suite: string): Promise<TestSuiteListing> {
-    return listingData[suite];
+  listing(suite: string): Promise<TestSuiteListing> {
+    return new Promise(resolve => {
+      resolve(listingData[suite]);
+    });
```

Comment:
nit:
```suggestion
    return Promise.resolve(listingData[suite]);
```

Reply:
Done. Oh nice! Didn't know about this syntax. Thanks!